packages/react/src/components/ButtonGroup/__stories__/ButtonGroup.stories.js

```diff
@@ -3,10 +3,11 @@ import { storiesOf } from '@storybook/react';
 import { withKnobs, text, select } from '@storybook/addon-knobs';
 import { ArrowRight20, ArrowDown20, Pdf20 } from '@carbon/icons-react';
 import '@carbon/grid/scss/grid.scss';
```

Comment:
@kennylam should this be added to the `index.scss` file too?

Reply:
Doh, yep I missed that one. Thanks @jeffchew